tests/test_terraform.py

```diff
@@ -45,12 +46,15 @@ def test_happy_path(self):
             outputs = executor.output()
             self.assertIsNotNone(outputs["vpc_id"])
             self.assertEqual(1, len(outputs["private_subnet_ids"]))
-            self.assertEqual(0, len(outputs["public_subnet_ids"]))
+            self.assertEqual(1, len(outputs["public_subnet_ids"]))
         finally:
             executor.destroy()
 
 
 class TestDeployClusterUsingCloudQ(unittest.TestCase):
+    QUMULO_CLUSTER_VERSION = "5.1.0.1"
```

Comment:
Will this need to be updated every time a Qumulo version is released? Would it be better to check the major version only here?